RockMigrations.module.php

```diff
@@ -194,8 +194,12 @@ public function down($version) {
    * @return void
    */
   public function executeInstall() {
-    $version = self::getModuleInfo()['version'];
-    return $this->executeUpgrade(null, $version);
+  	// check if module is set
```

Comment:
I've just replaced tab by spaces and testing the comment feature :)

Reply:
Cool, thanks! Have you thought about adding an `.editorconfig` file (https://editorconfig.org/) for your modules? There's one in the PW wire directory which contains this:

```ini
[*.{php,inc,module}]
indent_style = tab
trim_trailing_whitespace = false
insert_final_newline = true
```

But you can changes tab to space if you're that way inclined ;)

Reply:
Thx :) So how would that work exactly? I know that PW uses tabs, but I prefer spaces (don't know why). So how could I keep my preference while being compliant with PW style?

Reply:
The idea is that when using an editor that supports it - it will change the style/behaviour for the files targetted by .editorconfig files. So when editing files in /wire/ (and within), it will use tabs; and if you have a .editorconfig file in your module that specifies spaces - the editor should use spaces for files within your module :)

(I was just pointing to the PW one as an example as what's possible, not specifically about being 'compliant' with PW style in general or suggestion you should match it. You can have whatever style you want; and that this might help keep it if/when you take outside contributions :) )

Reply:
Ah, so that means everybody that contributes and whos' editor supports it will automatically use spaces for my modules? If yes, then I got it wrong in the first place, because I thought it would be the other way round: The file should have (pw style) tabs and my editor views them as spaces. That felt weird, that's why I asked :)

Reply:
Yes, that's right :) It's a bit of a hint to say "the author prefers things in this style" and everyone involved can adhere to it automatically (with supported editors :))

Reply:
Thank you - updated most of my modules :)

Reply:
Great! And thank _you_ again for your great modules 🙌 😃

Reply:
Thx! And thank you again for the PR ;)